### PR TITLE
New tekton.dev Experience (PR 1 of 2: Added Baseline Template)

### PR DESCRIPTION
**Pull request by michaelawyu**
# Changes

This PR removes the old tekton.dev site and added the
[example docsy website](https://github.com/google/docsy-example) as a baseline
template for future updates.

This PR, as discussed with the team, is the first one of two PRs that implement the new tekton.dev experience. This PR simply applies an already-reviewed Hugo theme to the website and includes only placeholder information, so as to fast track the review process.

Some contents in the template have been removed to reduce the size of the PR.
Contents removed include:

* Norwegian contents and settings for showcasing multi-language support
* Sample documentation for docsy usage

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [N/A] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [Done] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by michaelawyu** [comment]
@bobcatfish PTAL.

**Comment by carlos-logro** [comment]
Im assuming this isn't the PR that should be reviewed given that is just the Hugo framework right?

**Comment by michaelawyu** [comment]
@carlos-logro Yeah, this PR includes only the framework-related stuff and the theme.

File: code-of-conduct.md [hunk 1]
```diff
@@ -1,75 +0,0 @@
-# Contributor Covenant Code of Conduct
```
Comment (bobcatfish) [inline code-of-conduct.md]: we definitely don't want to delete this file

File: config.toml [hunk 2]
```diff
@@ -1,21 +1,150 @@
-title        = "Tekton Pipelines"
-baseURL      = "https://tekton-dev.netlify.com"
+baseURL = "/"
+title = "Goldydocs"
```
Comment (bobcatfish) [inline config.toml]: not sure if we want these 2 lines to change

File: config.toml [hunk 3]
```diff
@@ -1,21 +1,150 @@
...
-The **Tekton Pipelines** project provides [Kubernetes](https://kubernetes.io)-style resources for declaring CI/CD-style pipelines.
-"""
-comingsoon = "This site is currently under construction. For more information on the Tekton Pipelines project in the meantime, check out the [repo on GitHub](https://github.com/tektoncd/pipeline)."
-
-[params.logos]
-hero = "horizontal-color"
-
-[params.fonts]
-sansserif = "Franklin Gothic Heavy Regular"
+copyright = "The Docsy Authors"
```
Comment (bobcatfish) [inline config.toml]: the copyright should be "The Tekton Authors"

File: config.toml [hunk 4]
```diff
@@ -1,21 +1,150 @@
...
+	name = "Slack"
+	url = "https://example.org/slack"
+	icon = "fab fa-slack"
+        desc = "Chat with other project developers"
+[[params.links.developer]]
+	name = "Developer mailing list"
+	url = "https://example.org/mail"
+	icon = "fa fa-envelope"
+        desc = "Discuss development issues around the project"
+
```
Comment (bobcatfish) [inline config.toml]: a bunch of the values above can probably be removed in this first iteration since they are specific to another project

**Comment by bobcatfish** [comment]
p.s. I've added you as an owner on our netlify instance @michaelawyu but please hold back on making any changes for now - this will give you access to logs etc., but we should make sure to understand the existing setup before we try to make any changes

File: config.toml [hunk 5]
```diff
@@ -1,21 +1,115 @@
...
+yes = 'Thanks! <a href="https://github.com/tektoncd/website/issues/new">Tell us how we can further improve</a>.'
+no = 'Sorry about that. <a href="https://github.com/tektoncd/website/issues/new">Tell us how we can further improve</a>.'
+
+[params.links]
+# Developer relevant links. These will show up on right side of footer and in the community page if you have one.
+[[params.links.developer]]
+	name = "GitHub"
+	url = "https://github.com/tektoncd"
+	icon = "fab fa-github"
+        desc = "GitHub organization for the Tekton project"
```
Comment (bobcatfish) [inline config.toml]: i think the indentation here might be a bit off :thinking:

File: config.toml [hunk 6]
```diff
@@ -1,21 +1,115 @@
...
+
+# First one is picked as the Twitter card image if not set on page.
+# images = ["images/project-illustration.png"]
+
+# Menu title if your navbar has a versions selector to access old versions of your site.
+# This menu appears only if you have at least one [params.versions] set.
+version_menu = "Releases"
+
+# Repository configuration (URLs for in-page links to opening issues and suggesting changes)
+github_repo = "https://github.com/tektoncd/"
```
Comment (bobcatfish) [inline config.toml]: i think we want this to be `https://github.com/tektoncd/website` it looks like it's used on pages like https://deploy-preview-14--tekton-dev.netlify.com/docs/ at the bottom to link back to here?

File: config.toml [hunk 7]
```diff
@@ -1,21 +1,115 @@
...
+
+# Adds a H2 section titled "Feedback" to the bottom of each doc. The responses are sent to Google Analytics as events.
+# This feature depends on [services.googleAnalytics] and will be disabled if "services.googleAnalytics.id" is not set.
+# If you want this feature, but occasionally need to remove the "Feedback" section from a single page,
+# add "hide_feedback: true" to the page's front matter.
+[params.ui.feedback]
+enable = true
+# The responses that the user sees after clicking "yes" (the page was helpful) or "no" (the page was not helpful).
+yes = 'Thanks! <a href="https://github.com/tektoncd/website/issues/new">Tell us how we can further improve</a>.'
+no = 'Sorry about that. <a href="https://github.com/tektoncd/website/issues/new">Tell us how we can further improve</a>.'
```
Comment (bobcatfish) [inline config.toml]: neat!

File: content/en/_index.html [hunk 8]
```diff
@@ -0,0 +1,16 @@
++++
+title = "Goldydocs"
+linkTitle = "Goldydocs"
```
Comment (bobcatfish) [inline content/en/_index.html]: i think maybe these should both be "tekton" as well? doesnt seem to make much difference but it does show up in the page source

**Comment by michaelawyu** [comment]
All done.  @bobcatfish PTAL.

**Comment by bobcatfish** [comment]
Nice!!! Thanks for fixing it up :D

/lgtm

image

**Comment by bobcatfish** [comment]
Quick update: looks like tekton.dev is still pointing at firebase and not at netfliy. But that means we can keep updating https://tekton-dev.netlify.com/ and then switch over the DNS once we're ready :D :tada:

**Pull request merged**